plugins/labhub.py

```diff
@@ -278,7 +278,7 @@ def difficulty_level(user, iss):
                 1. A newcomer is asking for assignment to low or newcomer issue.
                 2. The user belongs to developers or maintainers team as well as
                    newcomers team.
-                False if
+                False if:
```

Comment:
Unrelated change; do it in a new commit.

Reply:
bump :+1:

Reply:
Created a newcomer issue about this https://github.com/coala/corobo/issues/520

Reply:
bump